src/core/temperaments/GOTemperament.h

```diff
@@ -16,25 +16,20 @@ class GOTemperament {
   wxString m_GroupTitle;
   wxString m_Name;
   wxString m_Title;
+  bool m_IsTemperamentOriginalBased; // To know how tuning should be handled
 
 public:
   GOTemperament(wxString name, wxString group = wxEmptyString);
   GOTemperament(
     wxString name, wxString title, wxString group, wxString groupTitle);
   virtual ~GOTemperament();
 
-  virtual float GetOffset(
-    bool ignorepitch,
-    unsigned midi_number,
-    unsigned wav_midi_number,
-    float wav_pitch_fract,
-    float harmonic_number,
-    float pitch_correction,
-    float default_tuning) const;
+  virtual float GetOffset(unsigned index) const;
   wxString GetName() const;
   wxString GetTitle() const;
   wxString GetGroup() const;
   wxString GetGroupTitle() const;
+  bool GetIsTemperamentOriginalBased() const;
```

Comment:
```suggestion
  bool IsTemperamentOriginalBased() const { return m_IsTemperamentOriginalBased; }
```

1. The `Is` prefix is sufficient for boolean getters;
2. The method is quite simple for inlining.